external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -422,21 +426,57 @@ private[kafka010] class KafkaOffsetReader(
 }
 
 private[kafka010] object KafkaOffsetReader {
+  type Record = ConsumerRecord[Array[Byte], Array[Byte]]
```

Comment:
I'd say the object `KafkaOffsetReader` is going to become like utility which everything in it are not used in companion class (weird), but `kafkaSchema` also seems to be in wrong place, so well... Let's hear voices on this.